demand-capacity-mgmt-frontend/.dockerignore

```diff
@@ -0,0 +1 @@
+/node_modules/.cache/.eslintcache
```

Comment:
I think you should ignore the complete node_modules directory, since you are anyways doing a [`yarn install`](https://github.com/eclipse-tractusx/demand-capacity-mgmt/blob/afc92e5471a4ebf6ba76d74f0498b1eb5d1d6909/build/frontend/Dockerfile#L33)

Reply:
Resolved